armaclass/parser.py

```diff
@@ -215,6 +216,10 @@ def parseProperty(self, context):
 
             value = self.parseArray()
 
+        elif current == SEMICOLON:
+            value = self.dict()
+            self.parseWhitespace()
+
```

Comment:
Any chance you could add a test for this case? I'm slowly starting to think about merging this ;)

Reply:
*ahem!* ;)